src/overlays/actors/ovl_En_Xc/z_en_xc.c

```diff
@@ -336,9 +349,11 @@ s32 EnXc_BoleroCS(EnXc* this, PlayState* play) {
 }
 
 void EnXc_SetupSerenadeAction(EnXc* this, PlayState* play) {
-    // Player is adult and does not have iron boots and has not learned Serenade
-    if (!CHECK_OWNED_EQUIP(EQUIP_TYPE_BOOTS, EQUIP_INV_BOOTS_IRON) && !GET_EVENTCHKINF(EVENTCHKINF_52) &&
+    // Player is adult and has not learned Serenade and, if in debug mode, does not have iron boots
```

Comment:
comment can prob be removed, this was from a time before `CHECK_OWNED_EQUIP` and `GET_EVENTCHKINF` even existed (the flag will be named at some point)